packages/devtools_app/test/inspector_v2/inspector_integration_test.dart

```diff
@@ -482,3 +481,41 @@ Future<void> _loadInspectorUI(WidgetTester tester) async {
     await tester.pumpAndSettle();
   }
 }
+
+Finder findNodeMatching(String text) => find.ancestor(
+      of: find.richTextContaining(text),
+      matching: find.byType(DescriptionDisplay),
+    );
+
+Finder findExpandCollapseButton(Finder nodeFinder) => find.descendant(
+      of: nodeFinder,
+      matching: find.byType(TextButton),
+    );
+
+Finder findExpandCollapseButtonText({
+  required Finder buttonFinder,
+  required String buttonText,
+}) =>
+    find.descendant(
+      of: buttonFinder,
+      matching: find.text(buttonText),
+    );
```

Comment:
nit: since these only appear to be called within the context of the method below, I recommend using these finders directly in the next method instead of creating two additional wrapper methods. This may help reduce confusion about which helper method to use moving forward when people are trying to add tests to this file

Reply:
Sure, moved them directly into the method